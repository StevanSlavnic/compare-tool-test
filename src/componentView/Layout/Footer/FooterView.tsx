import React from 'react'

export const FooterView = () => {
  return <>Footer</>
}
