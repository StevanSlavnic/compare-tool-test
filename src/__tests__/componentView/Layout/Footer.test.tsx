import React from 'react'
import { shallow } from 'enzyme'
import { FooterView } from 'componentView/Layout/Footer/FooterView'

it('renders Footer', () => {
  const component = shallow(<FooterView />)
  expect(component).toMatchSnapshot()
})
