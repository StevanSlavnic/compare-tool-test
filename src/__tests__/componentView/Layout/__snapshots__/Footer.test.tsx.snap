// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer 1`] = `ShallowWrapper {}`;
